app/src/main/java/com/example/awesomelanguagelearning/core/domain/models/User.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.awesomelanguagelearning.core.domain.models
+
+data class User(
+    val firstName: String = "",
+    val lastName: String = "",
+    val email: String = "",
+    val password: String = "",
+    val confirmPassword: String = "",
+    val isActive: Boolean = false
+) {
+    val arePasswordsCorrect: Boolean =
+        password.isNotBlank() && confirmPassword.isNotBlank()
+
+    val isUserDataCorrect: Boolean = email.isNotBlank()
+            && firstName.isNotBlank()
+            && lastName.isNotBlank()
+
+    val arePasswordsNotEqual: Boolean =
+        password != confirmPassword
```

Comment:
@igel911 I think it's a bad design of the class. Why do you assign all these fields if they will be initialized with class creation? It's better to make them as methods to avoid some unexpected behavior

Reply:
@slothdeboss done. Just done. I ran out of funny comments